In [1]:
# Make the import
from scipy.stats import norm

# Make the function call
norm.ppf(.9, loc=1.25, scale=.46)

# This function returns the value 1.8395137201505163

1.8395137201505163

The result tells us that any driver with a response time of approx. 1.84 seconds has a longer response time than 90% of the population.